jujugui/static/gui/src/app/components/charmbrowser/test-charmbrowser.js

```diff
@@ -28,15 +28,21 @@ describe('Charmbrowser', function() {
 
   beforeAll(function(done) {
     // By loading this file it adds the component to the juju components.
-    YUI().use('charmbrowser-component', function() { done(); });
+    YUI().use([
+      'charmbrowser-component',
+      'entity-details',
+      'mid-point',
+      'search-results'
```

Comment:
Because `React.createElement` (transpiled from the JSX in charmbrowser.js) complains (logs a warning) when the first param passed in is null. You don't see these warnings when running in PhantomJS; just in a real browser's console. `createElement` is still getting called even with the shallow renderer.